modules/ROOT/partials/configuration/end_container_on_empty_block.adoc

```diff
@@ -1,20 +1,20 @@
 [[end_container_on_empty_block]]
 == `+end_container_on_empty_block+`
 
-This option allows you to to split the current container block element if the enter key is pressed inside an empty inner block element.
+This option allows you to split the current container block element and specify the type of element created if the enter key is pressed inside an empty inner block element.
 
-*Type:* `+Boolean+`
+*Type:* `+Boolean+` or `+String+`
 
-*Default value:* `+false+`
+*Default value:* `+'blockquote'+`
 
-*Possible values:* `+true+`, `+false+`
+*Possible values:* `+true+`, `+false+`, `+String+`
```

Comment:
This isn't correct... it's not any string, it's a comma separated list of element names.
```suggestion
*Possible values:* `+true+`, `+false+` or a comma separated `+String+` of element tag names
```